{"name": "创建英雄失败的测试用例-攻击力不符合要求", "status": "passed", "parameters": [{"name": "success_for_name", "value": "'807834167'"}, {"name": "success_for_volume", "value": "2"}, {"name": "fail_for_power", "value": "-50"}], "start": 1684150563128, "stop": 1684150563128, "uuid": "30d23198-ceda-4312-af8f-8a43fd4ad82d", "historyId": "dba372e59bf92a8c377309294bd83d2d", "testCaseId": "052da0c40443119f715fb4201d430dc8", "fullName": "test_create_hero_homework_01.TestHeroCreate#test_create_hero_fail_for_power", "labels": [{"name": "tag", "value": "run(order=4)"}, {"name": "suite", "value": "test_create_hero_homework_01"}, {"name": "subSuite", "value": "TestHeroCreate"}, {"name": "host", "value": "LAPTOP-0EHG7NSH"}, {"name": "thread", "value": "48660-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_create_hero_homework_01"}]}